app/endpoints/users.py

```diff
@@ -303,9 +303,10 @@ async def create_user(
             content=f"Please confirm your MyECL account by using the following token in the application : {activation_token}",
```

Comment:
This can be changed, since Titan does now support path params for creating account and recovering password

Reply:
New PR?
We could also edit the mail content using better templates, to make easier the sign up for users.

Reply:
Yes, it could be really cool to update the whole mail template !